src/content/en/2021/compression.md

```diff
@@ -1,20 +1,271 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Compression
-description: TODO
+description: Compression chapter of the 2021 Web Almanac covering HTTP compression, algorithms, compression levels, content types, 1st party and 3rd party compression and opportunities.
 authors: [lvandeve, mo271, jyrkialakuijala]
 reviewers: [fischbacher, eustas]
 analysts: [paulcalvano]
 editors: []
 translators: []
+jyrkialakuijala_bio: Jyrki Alakuijala is an active member of the open source software community, and a data compression researcher. Jyrki works at Google as a Technical Lead/Manager, and his recent published work has been with Zopfli, Butteraugli, Guetzli, Gipfeli, WebP lossless, Brotli, and JPEG XL compression formats and algorithms, and two hashing algorithms, CityHash, and HighwayHash. Before his Google employment he developed software for neurosurgery and radiation therapy treatment planning.
+mo271_bio: Moritz Firsching is software engineer at Google Switzerland, where he works on progressive image formats and font compression. Before that Moritz did research as a mathematician studying polytopes.
```

Comment:
Have you checked with Moritz that this is a good description? I would say he's done (and is still doing) quite a few other things as well in mathematics.

Reply:
Yes, Moritz is happy with the bio